llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -1191,8 +1191,7 @@ void SPIRVEmitIntrinsics::processInstrAfterVisit(Instruction *I,
       B.SetInsertPoint(I);
       Value *OpTyVal = Op;
       if (Op->getType()->isTargetExtTy())
-        OpTyVal = Constant::getNullValue(
-            IntegerType::get(I->getContext(), GR->getPointerSize()));
+        OpTyVal = UndefValue::get(Op->getType());
```

Comment:
Prefer PoisonValue::get() unless you need "undef" for some specific reason.

Reply:
Thank you for the suggestion, applied.